File: Modul6/Soal1/PRAK501-2210817110008-Aditya Oktaviari.c
#include <stdio.h>

int MaxBilangan(int a, int b, int c, int d)
{
    int max;
    if(a>b && a>c && a>d){
        max=a;
    }
    else if(b>a && b>c && b>d){
        max=b;
    }
    else if(c>b && c>a && c>d){
        max=c;
    }
    else if(d>b && d>c && d>a){
        max=d;
    }
}

int main() {
int a, b, c, d;
scanf("%d %d %d %d", &a, &b, &c, &d);
int hasil = MaxBilangan(a, b, c, d);
printf("%d", hasil);
return 0;
}

File: Modul6/Soal1/PRAK501-2210817110008-Aditya Oktaviari.py
def MaxBilangan(a, b, c, d):
    if(a>b and a>c and a>d):
        return a
    
    elif(b>a and b>c and b>d):
        return b
    
    elif(c>b and c>a and c>d):
        return c
    
    elif(d>b and d>c and d>a):
        return d
    
a,b,c,d = input().split()
hasil = int(MaxBilangan(a, b, c, d))
print("%d" % (hasil))

